notifier/chatters.rst

```diff
@@ -306,6 +306,27 @@ to add `message options`_::
 
     $chatter->send($chatMessage);
 
+Updating Telegram messages
```

Comment:
```suggestion
Updating Telegram Messages
```

(our title case is a bit confusing, but every "non-closed word" must be capitalized)